CRM/CiviGeometry/Tasks.php

```diff
@@ -48,8 +48,7 @@ public static function geoplaceAddress(CRM_Queue_TaskContext $ctx, $address_id)
    * Get all the Addresses for this geometry
    */
   public static function buildGeometryRelationships(CRM_Queue_TaskContext $ctx, $geometry_id) {
-    $matches = CRM_CiviGeometry_BAO_Geometry::getAddresses($geometry_id);
-    foreach ($matches as $match) {
+    foreach (CRM_CiviGeometry_BAO_Geometry::getAddresses($geometry_id, []) as $match) {
```

Comment:
is there a reason why you need the [] if that is the default value?

Reply:
Nope, it was because in the other code I adapted this from (to run this manually, specifically for large thing in WA) I was using non-default values and just removed them. I'll change that.